src/hexdoc/_templates/components/table_of_contents.html.jinja

```diff
@@ -14,10 +17,7 @@
 
   {% for category in book.categories.values() if category.entries.values() %}
     <details class="toc-category">
-      {# category #}
```

Comment:
nice - I guess I forgot to remove these, they're really not necessary now. thanks!